DatadogSessionReplay/Sources/Recorder/RecordingCoordinator.swift

```diff
@@ -157,6 +185,10 @@ internal class RecordingCoordinator {
         telemetry.stopMethodCalled(methodCalledTrace, isSuccessful: isSuccessful)
     }
 
+    private enum Constants {
+        static let throttlingRate: TimeInterval = 0.1 // 100ms
```

Comment:
/question Why changing from [config](https://github.com/DataDog/dd-sdk-ios/pull/2074/files#diff-dde69db27680d1c31b894abbb9a1ddc5009ecb3a51e555a112302faed0306834L52) to constant?